api/specs/web-server/_resource_usage.py

```diff
@@ -87,7 +87,7 @@ async def export_resource_usage_services(
     order_by: Annotated[
         Json | None,
         Query(
-            description="Order by field (started_at|stopped_at|credit_cost) and direction (asc|desc). The default sorting order is ascending.",
+            description="Order by field (wallet_id|wallet_name|user_id|project_id|project_name|node_id|node_name|service_key|service_version|service_type|started_at|stopped_at|service_run_status|credit_cost|transaction_status) and direction (asc|desc). The default sorting order is ascending.",
```

Comment:
MINOR: The descriptions are particularly useful in the OAS. This is a long description and I would create a constant in `api/specs/web-server/_resource_usage.py` and use it there and in `services/web/server/src/simcore_service_webserver/resource_usage/_service_runs_handlers.py`